dags/nps_pilotage.py

```diff
@@ -0,0 +1,57 @@
+from airflow import DAG
+from airflow.decorators import task
+from airflow.models import Variable
+from airflow.operators import empty
+
+from dags.common import dates, db, default_dag_args, slack
+
+
+with DAG(
+    "nps_pilotage",
+    schedule_interval="@weekly",
+    **default_dag_args(),
+) as dag:
+    start = empty.EmptyOperator(task_id="start")
+
+    end = slack.success_notifying_task()
+
+    @task(task_id="create_nps")
+    def create_nps(**kwargs):
+        import numpy as np
+        import pandas as pd
+        from sqlalchemy import create_engine
+
+        df_int = []
+
+        for name, pub_sheet_url in Variable.get("NPS_DASHBOARD_PILOTAGE", deserialize_json=True):
+            print(f"reading {name=} at {pub_sheet_url=}")
+            sheet_df = pd.read_csv(pub_sheet_url)
+            sheet_df["Submitted at"] = pd.to_datetime(sheet_df["Submitted at"], dayfirst=True)
+            sheet_df.rename(
+                columns={
+                    "Submitted at": "Date",
+                    "Quelle est la probabilité que vous recommandiez ce tableau de bord à un collègue, "
+                    "partenaire ou homologue ?": "Recommendation",
+                },
+                inplace=True,
+            )
+            sheet_df["Nom Du Tb"] = name
+            sheet_df = sheet_df[["Recommendation", "Nom Du Tb", "Date"]]
+            df_int.append(sheet_df)
+        df = pd.concat(df_int)
+
+        # I need to create these columns and fill them with NaN
```

Comment:
`I` dans le code ? Comment sait-on qui est `I` ? On doit aller faire un `git blame` ? Soit tu mets ton nom/pseudo, soit tu mets `We`. Ou encore `These columns need to be created` sans pronom.